test/integration/vaultauthmethods/terraform/variables.tf

```diff
@@ -83,3 +83,41 @@ variable "operator_image_tag" {
 variable "auth_role" {
   default = "role1"
 }
+
+variable "irsa_assumable_role_arn" {
+  default = ""
+}
+
+variable "aws_region" {
+  default = "us-east-2"
+}
+
+variable "aws_account_id" {
+  default = ""
+}
+
+variable "aws_access_key_id" {
+  description = "AWS_ACCESS_KEY_ID for testing static creds with AWS auth"
+  default     = ""
+}
+
+variable "aws_secret_access_key" {
+  description = "AWS_SECRET_ACCESS_KEY for testing static creds with AWS auth"
+  default     = ""
+  sensitive   = true
+}
+
+variable "aws_static_creds_role" {
```

Comment:
Is there any way we could create the required variables here in the infra/eks folder or perhaps create-eks make cmd?

Trying to reduce some of the pre-requisites before running the integration-test-eks